CONTRIBUTING.md

```
@@ -224,35 +250,40 @@
 
 Great you've seen the coverage report and have identified areas for improvement! Here are instructions for testing the bot.
 
-### Testing the Bot Locally
+### Validating Changes to the Bot Locally
 
-Since we currently don't have a hosting solution, you'll need to run the bot locally to test your changes. Here's how to do it:
+#### Validate in Development BEFORE pushing to stage
 
-1. Make sure you have all the required dependencies installed:
+We use separate Discord bots for each environment:  
+- **Development Bot** (`Bug [Development]`)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)